sonar-orchestrator/src/main/java/com/sonar/orchestrator/Orchestrator.java

```diff
@@ -142,7 +133,12 @@ public void start() {
 
   private void updateSetting(String key, String value) {
     if (getServer().version().isGreaterThanOrEquals("6.1")) {
-      server.adminWsClient().post("api/settings/set", ImmutableMap.of("key", key, "value", value));
+      server.newHttpCall("/api/settings/set")
+        .setMethod(HttpMethod.POST)
+        .setAdminCredentials()
+        .setParam("key", key)
+        .setParam("value", value)
+        .execute();
     } else {
       server.getAdminWsClient().update(new PropertyUpdateQuery(key, value));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "getAdminWsClient"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)